watch.json

```diff
@@ -0,0 +1,6 @@
+"restart": {
+    "include": [
+      "RoomMessages.js"
+    ],
+    "throttle": 100
```

Comment:
From [the example here](https://glitch.com/edit/#!/watch-json?path=watch.json:20:0).